README.md

```diff
@@ -31,217 +49,213 @@ wiki
 
-To create a page/tab that contains sub-pages, create a directory inside of [docs/](https://github.com/NDCLab/wiki/tree/gh-pages/docs) named after the page/tab you plan on creating.
+## Adding Sections
 
-For example, to add a tab called `A` on the wiki navbar, create directory `docs/A/`. 
+To add a brand new section on the wiki (and link it to the left-hand navigation bar), you need to do three things:
+1. create a new directory inside of `docs/` named after the new section
+2. within this new folder, create the base markdown file that controls the section
+3. also within this new folder, create one new markdown page for each content page that will live within this section of the wiki (see [Adding Pages](#adding-pages) section above for details).
 
-Next, create a markdown file inside of this directory and specify its position on the navbar and set the `has_children` tag to be `true`. 
+The base markdown file (#2 above) specifies the [slug](https://en.wikipedia.org/wiki/Clean_URL#Slug) that will be used to access all new pages in the section, as well as its position on the navbar. This markdown file should be named `[slug].md` and should begin with the following content:
 
 ```yml
 ---
 layout: default
 title: <title>
 nav_order: <navbar-ordinal>
 has_children: true
+permalink: docs/<slug>
 ---
 
 # <title>
 ```
 
-To create child pages located underneath this parent page, create a markdown file also located in this directory and specify the parent title in the header: 
+When adding a section, be careful to adjust the `nav_order` of any other navbar sections that will be impacted by the addition of a new section. This is accomplished by opening the appropriate markdown file(s) within the associated directory and changing the `nav_order` number in the header(s).
+
+##### Working Example:
+
+The **Around the Lab** tab on the wiki is placed in the fourth position in the navbar by specifying the following header in the `docs/around-the-lab.md` file: 
 
 ```yml
 ---
 layout: default
-title: <title>
-parent: <parent-title> 
-nav_order: <navbar-ordinal>
+title: Around the Lab
+nav_order: 4
+has_children: true
+permalink: /docs/around-the-lab
 ---
 
-# <title>
+# Around the Lab
 ```
 
-#### Working Example
+![readme_nav-example](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/technical/readme_nav-example.png)
 
-The *onboarding* tab on the wiki contains the child *accessing the hpc* in the navbar by specifying the following header in the [Onboarding/onboarding.md](https://github.com/NDCLab/wiki/blob/gh-pages/docs/Onboarding/onboarding.md) file:
+## Building Site on Local
 
-```yml
----
-layout: default
-title: Onboarding
-nav_order: 2
-has_children: true
-permalink: /docs/onboarding
----
+To view what the site would look like with the applied formatting and styles as seen on the live site, you can build a local copy by following the steps listed below. 
```

Comment:
OK, so this is the workaround to checking how the rendered images will look, right? If so, we should mention this above as well @F-said @jessb0t

Reply:
Actually, no...for this to work, one would have to hard-code the dev branch into the image link, then change to "main" before PR.  I think that it is fundamentally a discrepancy between Markdown and HTML (if I understood some of the Stack Overflow comments correctly).

Reply:
Added to devOps agenda.